util/lint_testplan.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This script lint the various opentitan testplan.hjson
+usage:
+    python3 ./util/lint_testplan.py --schema hw/lint/sival_testplan_schema.json \
+                                    --dir hw/top_earlgrey/data/ip/
+"""
+
+import argparse
+import glob
+import logging
+import sys
+
+import hjson
+import json
+import jsonschema
+
+parser = argparse.ArgumentParser()
```

Comment:
Makes sense to put `parser` in the `if __name`... block.

Ideally no code runs importing a python module

Reply:
done.